prometheus/doc/alert-manager.md

```diff
@@ -0,0 +1,53 @@
+OpenPai leverage [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to do
+alert. Alertmanager can send administrator an email to indicate cluster failure or problem with
```

Comment:
**wording tuning suggestions:**
Building on top of [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager/), OpenPAI started to support sending cluster failure and issues emails to administrator since 0.7 release.